File: RootTools2/src/main/java/com/rarnu/tools/neo/utils/ImageUtils.java
package com.rarnu.tools.neo.utils;

import android.graphics.Bitmap;

import java.io.File;
import java.io.FileOutputStream;

/**
 * Created by rarnu on 11/21/16.
 */
public class ImageUtils {

    public static void saveBitmapToFile(Bitmap bmp, String fileName) {
        if (fileName != null && !fileName.equals("")) {
            try {
                File f = new File(fileName);
                f.createNewFile();
                FileOutputStream fOut = null;
                fOut = new FileOutputStream(f);
                bmp.compress(Bitmap.CompressFormat.PNG, 100, fOut);
                fOut.flush();
                fOut.close();
            } catch (Exception e) {

            }
        }
    }

}

File: RootTools2/src/main/java/com/rarnu/tools/neo/utils/ImageUtils.kt
package com.rarnu.tools.neo.utils

import android.graphics.Bitmap

import java.io.File
import java.io.FileOutputStream

/**
 * Created by rarnu on 11/21/16.
 */
object ImageUtils {

    fun saveBitmapToFile(bmp: Bitmap?, fileName: String?) {
        if (fileName != null && fileName != "") {
            try {
                val f = File(fileName)
                f.createNewFile()
                val fOut = FileOutputStream(f)
                bmp?.compress(Bitmap.CompressFormat.PNG, 100, fOut)
                fOut.flush()
                fOut.close()
            } catch (e: Exception) {

            }

        }
    }

}
